package.json

```diff
@@ -31,6 +31,7 @@
     "babel-preset-es2015": "^6.14.0",
     "babel-preset-react": "^6.11.1",
     "babel-preset-stage-0": "^6.5.0",
+    "chokidar": "^1.6.0",
```

Comment:
We add this because yarn won't install this automatically.
Not sure who cause the bug.